ufl/domain.py

```diff
@@ -209,6 +209,9 @@ def ufl_coordinate_element(self):
     def ufl_cell(self):
         return self._ufl_cell
 
+    def ufl_meshes(self):
```

Comment:
This is a missing part of the API. I only added it so I wouldn't have to access a private attribute when stripping one of these objects. What I have done mirrors how private attributes are accessed throughout UFL.